news/2024/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!.md

```diff
@@ -0,0 +1,257 @@
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Utiba.jpeg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>
+
+What is your drawing process like? Is there a certain workflow you follow when working on each project, or does each project have a different process?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/13103233" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Dreamxiety.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/13103233">Dreamxiety</a></p>
+
+If I'm looking for ideas, I'll browse through the internet to find inspirations. For osu contests specifically, I often just search for art that has the theme. It's a pretty good method since what comes at the top of the search are the things people generally find appealing, and what people have in mind when the name of the theme comes up.
+
+Then I'll look for references, mainly for the things I'm not sure how to draw. Sometimes I also would use other images as placeholders for the background, like the sky for example. I'll adjust the colors and position, it helps me find what I'm trying to go for.
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/11354436" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Utiba.jpeg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/11354436">Utiba</a></p>
+
+Let's talk about osu!'s seasonal art contests, what made you interested back when you participated for the first time?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/13103233" style="background-image: url('/wiki/shared/news/2024-09-02-the-followpoint-dreamxiety-the-picasso-of-osu!/avatar-Dreamxiety.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/13103233">Dreamxiety</a></p>
+
+I remember that I was opening the osu! homepage and I saw the announcement for [2019 Halloween Contest](https://osu.ppy.sh/community/contests/81). Previously, I had no idea that osu! even hosts that so it was a surprise and I was hyped. I joined not expecting to get in but surprisingly I did and I was really happy.
+
+That was also the first time I got my supporter tag, which I thought was cool. Back then I've never got to enjoy any sort of paid perks in a game because I was just a broke student and my parents aren't a fan of that kinda stuff lol.
```

Comment:
```suggestion
That was also the first time I got my osu!supporter tag, which I thought was cool. Back then I've never got to enjoy any sort of paid perks in a game because I was just a broke student and my parents aren't a fan of that kinda stuff lol.
```